core-web/libs/portlets/edit-ema/ui/src/lib/dot-results-seo-tool/dot-results-seo-tool.component.ts

```diff
@@ -70,13 +70,29 @@ export class DotResultsSeoToolComponent implements OnInit, OnChanges {
     noFavicon = false;
 
     ngOnInit(): void {
+        const truncateText = (text: string, limit: number) => {
```

Comment:
This can be converted to a utility, and there add unit tests to check if you have all you need.